packages/experimental/TabList/src/TabList.types.ts

```diff
@@ -0,0 +1,113 @@
+import type * as React from 'react';
+import type { View } from 'react-native';
+
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import type { FocusZoneProps } from '@fluentui-react-native/focus-zone';
+import type { FontTokens, IForegroundColorTokens, IBackgroundColorTokens } from '@fluentui-react-native/tokens';
+
+export const tabListName = 'TabList';
+
+export interface TabListContextData {
+  /**
+   * The currently selected TabsItem's key
+   */
+  selectedValue: unknown;
```

Comment:
I don't think you should leave type as unknown. Would they be strings if they are keys?

Reply:
if they are TabItems, at least defined it and mark it with /* TODO */ tags

Reply:
> I don't think you should leave type as unknown. Would they be strings if they are keys?

This is a change that web made in their updated Tab component. `unknown` is used here because it's a more type-safe version of any. This would allow me to assign a tab's value to be a string, a number, an object, etc.

Reply:
On web, it has to be set to unknown because they are pltform specific HTML values. but for RN we type our props to be more specific. Also, use selectedKey instead of selectedValue (see Menu, RadioGroup for reference), and you can also follow the type convention there.